### PR TITLE
add a utility func. of getting standard parameters for hrh historical scaling and rescaling to effective capabilities in mode 2

### PR DESCRIPTION
**Pull request by BinglingICL**
This is created for scenario runs in mode 2 that require latest HRH capabilities by 2024.

### CODE REVIEW

**Comment by tbhallett** [comment]
Thanks @BinglingICL  - the utility function seems very good.

What's the change in the .xlsx file?

**Comment by BinglingICL** [comment]
> What's the change in the .xlsx file?

Thanks for checking Tim. The difference to master is that I have copied and added the sheet of "historical_scaling" from PR #1414, for adding the parameters of yearly scaling up factors.

File: resources/healthsystem/human_resources/scaling_capabilities/ResourceFile_dynamic_HR_scaling.xlsx
Comment (tbhallett) [inline resources/healthsystem/human_resources/scaling_capabilities/ResourceFile_dynamic_HR_scaling.xlsx:1]: @BinglingICL -- this file is now in master, so I think we can remove it from this PR?

**Review by tbhallett — CHANGES_REQUESTED** [review]

File: resources/healthsystem/human_resources/scaling_capabilities/ResourceFile_dynamic_HR_scaling.xlsx
Comment (BinglingICL) [reply resources/healthsystem/human_resources/scaling_capabilities/ResourceFile_dynamic_HR_scaling.xlsx:1]: Now master merged in and the resource file is the one in the master.

**Comment by tbhallett** [comment]
Excellent - thank you very much for this, @BinglingICL

**Review by tbhallett — APPROVED** [review]

**Pull request merged**